cime_config/config_component_cesm.xml

```diff
@@ -84,7 +88,7 @@
 
   <entry id="TIMER_LEVEL">
     <type>integer</type>
-    <default_value>4</default_value>
+    <default_value>12</default_value>
```

Comment:
Why?

Reply:
This is what cpl7 is using. I was trying to make the two more consistent. I am happy to revert back. Let me know what you think.